src/main/java/pong/Environment.java

```diff
@@ -1,8 +1,8 @@
 package pong;
 
 public class Environment {
-    public static final int WIDTH = 100;
-    public static final int HEIGHT = 40;
+    public static final int WIDTH = 700;
+    public static final int HEIGHT = 500;
```

Comment:
thank you for changing the width and height. this seems like good dimensions

Reply:
👍